lib/vagrant-service-manager/command.rb

```diff
@@ -183,7 +183,7 @@ def restart_service(service)
                     # to OpenShift service
                     "sccli #{service}"
```

Comment:
Agreed.  This needs to be implemented in vagrant-service-manager, not adb-utils.